querydsl-lucene5/src/main/java/com/querydsl/lucene5/AbstractLuceneQuery.java

```diff
@@ -44,11 +35,7 @@
 import org.jetbrains.annotations.Nullable;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Quite a few star imports here. Apparently they are not caught by Checkstyle, but observing that these mostly occur in older files, it does appear our convention is to avoid them.